views/js/qtiXmlRenderer/renderers/Container.js

```diff
@@ -55,6 +55,8 @@ define(['tpl!taoQtiItem/qtiXmlRenderer/tpl/container'], function(tpl){
                 function($0){
                     return $0.replace('</rt', '&nbsp;</rt');
                 });
+            // merge sibling tags strong, em, sub, sup
+            returnValue = mergeSiblings(encodedStr);
```

Comment:
```suggestion
            returnValue = mergeSiblings(returnValue);
```
because returnValue already contains some updates from previous replace

Reply:
Oh dear ... good catch!